db/migrations/000001_create_activities_table.up.sql

```diff
@@ -0,0 +1,25 @@
+BEGIN;
+
+CREATE TABLE activities
+(
+    id              bigserial       CONSTRAINT activities_pk PRIMARY KEY,
+    uuid            uuid            DEFAULT uuid_generate_v4() NOT NULL,
+    fork_id         uuid            NULL,
+    content_id      uuid            NOT NULL,
```

Comment:
We have adopted the pattern of storing `${field}_id  uuid` and are using the `uuid` property to reference the FK. 

With the aim of making it clearer what we are storing: _Should we rename this column to `${field}_uuid  uuid` or use the `id` property to reference the FK?_

@arianerocha @rluders

Reply:
Uhhh, good question, I think we should keep {field}_uuid as a convention to make it easier to identify the data type.
about using uuid as fk, I have no strong feelings. Normally using Id bigint sequential as FK makes searches faster in the long run. But we would have to see if we would have any tradeoff

Reply:
> about using uuid as fk, I have no strong feelings. Normally using Id bigint sequential as FK makes searches faster in the long run. But we would have to see if we would have any tradeoff

That was the idea of using `{field}_id` instead of using the `{field}_uuid`. AFAIK, we are not using uuid as FKs in our database. It can cause us some performance issues since uuid isn't sequential. I mean, unless we are using uuidv7.

I have no strong feelings here. We can move all to use uuid if we want, but then I would probably recommend we to check the uuidv7 or upper support for our code and database.

Reply:
I think we can continue using ID BigInt as FKs for links between entities that belong to the same MS and UUID as references for entities that belong to another MS, as well as the communication between the MSs.

Reply:
Woooooooorks for me.

Reply:
Considering what you said about id and uuid and reading some Postgres references, I believe that the best way would be to use `UUID V7` in our microservices. This way we have the advantage of identifying an object globally and do not lose performance. 

However, the implementation of V7 is not yet considered stable and in the uuid package we use I did not find information about it. I found this other package:

https://pganalyze.com/blog/5mins-postgres-uuid-vs-serial-primary-keys

[Postgres Extension](https://pgxn.org/dist/pg_uuidv7/#:~:text=Build%20using%20Docker-,pg_uuidv7%20%3A%20Use%20the%20new%20v7%20UUIDs%20in%20Postgres,version%207%20UUIDs%20in%20Postgres.)

[Docker - Postgres 16 with uuid_v7](https://github.com/fboulnois/pg_uuidv7?tab=readme-ov-file#build-using-docker)

[google/uuid/v7](https://pkg.go.dev/github.com/google/uuid@v1.6.0#NewV7)